sos/plugins/ovirt.py

```diff
@@ -85,7 +86,23 @@ def setup(self):
         self.add_forbidden_path('/etc/rhevm/.pgpass')
         # Copy all engine tunables and domain information
         self.add_cmd_output("engine-config --all")
-        self.add_cmd_output("engine-manage-domains list")
+
+        # 3.x line uses engine-manage-domains, 4.x uses ovirt-aaa-jdbc-tool
+        if is_executable('engine-manage-domains'):
+            self.add_cmd_output("engine-manage-domains list")
+        if is_executable('ovirt-aaa-jdbc-tool'):
+            cmd = 'ovirt-aaa-jdbc-tool'
+            subcmds = [
+                       'query --what=user',
+                       'query --what=group',
+                       'settings show'
+                       ]
+
+            self.add_cmd_output([
+                                 '%s %s' % (cmd, sub) for sub in subcmds
+                                 ]
+                                )
+
```

Comment:
You cannot easily list domains provided by AAA extensions, but you can list configured extensions using:
```
ovirt-engine-extensions-tool info list-extensions
```
which will produce following content (default installation, no aaa-ldap configured):
```
2017-09-07 11:19:50,789+02 INFO    ========================================================================
2017-09-07 11:19:50,806+02 INFO    ============================ Initialization ============================
2017-09-07 11:19:50,806+02 INFO    ========================================================================
2017-09-07 11:19:50,813+02 INFO    Loading extension 'internal-authz'
2017-09-07 11:19:50,815+02 INFO    Extension 'internal-authz' loaded
2017-09-07 11:19:50,816+02 INFO    Loading extension 'internal-authn'
2017-09-07 11:19:50,817+02 INFO    Extension 'internal-authn' loaded
2017-09-07 11:19:50,817+02 INFO    Initializing extension 'internal-authz'
2017-09-07 11:19:50,817+02 INFO    Extension 'internal-authz' initialized
2017-09-07 11:19:50,818+02 INFO    Initializing extension 'internal-authn'
2017-09-07 11:19:50,819+02 INFO    Extension 'internal-authn' initialized
2017-09-07 11:19:50,819+02 INFO    Start of enabled extensions list
2017-09-07 11:19:50,820+02 INFO    Instance name: 'internal-authn', Extension name: 'Internal Authn (Built-in)', Version: 'N/A', Notes: '', License: 'ASL 2.0', Home: 'http://www.ovirt.org', Author 'The oVirt Project', Build interface Version: '0',  File: '/home/mperina/ovirt-engine/etc/ovirt-engine/extensions.d/internal-authn.properties', Initialized: 'true'
2017-09-07 11:19:50,820+02 INFO    Instance name: 'internal-authz', Extension name: 'Internal Authz (Built-in)', Version: 'N/A', Notes: '', License: 'ASL 2.0', Home: 'http://www.ovirt.org', Author 'The oVirt Project', Build interface Version: '0',  File: '/home/mperina/ovirt-engine/etc/ovirt-engine/extensions.d/internal-authz.properties', Initialized: 'true'
2017-09-07 11:19:50,820+02 INFO    End of enabled extensions list
2017-09-07 11:19:50,821+02 INFO    ========================================================================
2017-09-07 11:19:50,821+02 INFO    ============================== Execution ===============================
2017-09-07 11:19:50,821+02 INFO    ========================================================================
2017-09-07 11:19:50,825+02 INFO    Extension instance='internal-authn' name='Internal Authn (Built-in)' version='N/A'
2017-09-07 11:19:50,825+02 INFO    Extension instance='internal-authz' name='Internal Authz (Built-in)' version='N/A'
```

Using `ovirt-aaa-jdbc-tool` with above commands IMO reveals too much sensitive informations about customer setup and don't cover other extensions like aaa-ldap